src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2986,16 +2988,19 @@ float Creature::GetAggroRange(Unit const* target) const
     if (aggroRate == 0)
         return 0.0f;
 
-    uint32 targetLevel = target->getLevelForTarget(this);
-    uint32 myLevel = getLevelForTarget(target);
-    int32 levelDiff = int32(targetLevel) - int32(myLevel);
+    uint32 creaturelevel = target->getLevelForTarget(this);
+    uint32 playerlevel  = getLevelForTarget(target);
+    int32 levelDiff = int32(creaturelevel) - int32(playerlevel);
 
     // The maximum Aggro Radius is capped at 45 yards (25 level difference)
     if (levelDiff < -25)
         levelDiff = -25;
 
     // The base aggro radius for mob of same level
-    float aggroRadius = 20.0f;
+    float const detectionRange = GetDetectionRange();
+    float aggroRadius = detectionRange;
+    if (aggroRadius < 1)
+        return 0.0f;
```

Comment:
https://www.azerothcore.org/wiki/cpp-code-standards#brackets

Reply:
This struck me as odd when you don't use them in the file
![image](https://user-images.githubusercontent.com/5585555/122463678-2e236080-cfb6-11eb-91c2-2836665e4a59.png)

Reply:
```cpp
if (aggroRadius &lt; 1)
{
    return 0.0f;
}

Reply:
do you really want to have &lt; instead of < ?
that's not a very good solid

Reply:
No, it was weird copy shit on chrome. I want the brackets

Reply:
done